tests/scripts/thread-cert/v1_2_LowPower_5_3_01_SSEDAttachment.py

```diff
@@ -0,0 +1,149 @@
+#  AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+#  IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+#  ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+#  LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+#  CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+#  SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+#  INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+#  CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+#  ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+#  POSSIBILITY OF SUCH DAMAGE.
+#
+
+import unittest
+
+from config import ADDRESS_TYPE
+from mac802154 import MacHeader
+from pktverify import consts
+from pktverify.packet_verifier import PacketVerifier
+
+import thread_cert
+
+LEADER = 1
+ROUTER = 2
+SSED_1 = 3
+
+
+class LowPower_5_3_01_SSEDAttachment(thread_cert.TestCase):
+    TOPOLOGY = {
+        LEADER: {
+            'version': '1.2',
+            'name': 'LEADER',
+            'mode': 'rsdn',
+            'panid': 0xface,
+            'allowlist': [ROUTER, SSED_1]
+        },
+        ROUTER: {
+            'version': '1.2',
+            'name': 'ROUTER',
+            'mode': 'rsdn',
+            'panid': 0xface,
+            'router_selection_jitter': 1,
+            'allowlist': [LEADER],
+        },
+        SSED_1: {
+            'version': '1.2',
+            'name': 'SSED_1',
+            'mode': 's',
+            'panid': 0xface,
+            'allowlist': [LEADER],
+        },
+    }
+
+    def test(self):
+        self.nodes[LEADER].start()
+        self.simulator.go(5)
+        self.assertEqual(self.nodes[LEADER].get_state(), 'leader')
+
+        self.nodes[ROUTER].start()
+        self.simulator.go(5)
+        self.assertEqual(self.nodes[ROUTER].get_state(), 'router')
+
+        self.nodes[SSED_1].set_csl_period(consts.CSL_DEFAULT_PERIOD)
+        self.nodes[SSED_1].start()
+        self.simulator.go(5)
+        self.assertEqual(self.nodes[SSED_1].get_state(), 'child')
+
+        self.simulator.go(5)
+
+        self.collect_rloc16s()
+        self.collect_rlocs()
+
+        # Verify connectivity by sending an ICMP Echo Request from the Router to SSED_1 mesh-local address via the DUT
+        # Set ping timeout as two CSL periods.
+        timeout = 2 * consts.CSL_DEFAULT_PERIOD * consts.US_PER_TEN_SYMBOLS / 1000000
+        self.assertTrue(self.nodes[ROUTER].ping(self.nodes[SSED_1].get_ip6_address(ADDRESS_TYPE.RLOC),
+                                                timeout=timeout))
+
+        # Verify fragmented CSL transmission
+        self.assertTrue(self.nodes[ROUTER].ping(self.nodes[SSED_1].get_ip6_address(ADDRESS_TYPE.RLOC),
+                                                size=128,
+                                                timeout=timeout))
```

Comment:
one fragment should occupy one csl period, right? If so, should here increases the timeout?